commit 6ac58139ce8d2f30f43ba787c40c65e6bb8cdf29
Author: q66 <daniel@octaforge.org>
Date:   Tue Nov 8 17:46:35 2022 +0000

    move SAMBA_CHECK_UNDEFINED_SYMBOL_FLAGS to the end
    
    if not, any compiler checks done afterwards will for some reason
    hang the configure on ppc32 forever, which we do not want

diff --git a/wscript b/wscript
index 262f1bf..1688fc5 100644
--- a/wscript
+++ b/wscript
@@ -356,8 +356,6 @@ def configure(conf):
     conf.RECURSE('lib/mscat')
     conf.RECURSE('packaging')
 
-    conf.SAMBA_CHECK_UNDEFINED_SYMBOL_FLAGS()
-
     # gentoo always adds this. We want our normal build to be as
     # strict as the strictest OS we support, so adding this here
     # allows us to find problems on our development hosts faster.
@@ -397,6 +395,8 @@ def configure(conf):
 
     conf.SAMBA_CONFIG_H('include/config.h')
 
+    conf.SAMBA_CHECK_UNDEFINED_SYMBOL_FLAGS()
+
 def etags(ctx):
     '''build TAGS file using etags'''
     from waflib import Utils
